go/billing/tests/test_tasks.py

```diff
@@ -438,7 +585,7 @@ def test_generate_monthly_statement_unaccessible_tags(self):
         statement = tasks.generate_monthly_statement(
             self.account.id, *this_month())
 
-        [item] = get_line_items(statement).filter(channel=u'tag2.1')
+        item = get_line_items(statement).filter(channel=u'tag2.1').latest('id')
```

Comment:
We are now getting a line item for messages received and one for session length, this change is to only check one of them, since that seems to sufficient for this test.